x/ibc/applications/transfer/types/keys.go

```diff
@@ -30,6 +29,10 @@ const (
 
 	// DenomPrefix is the prefix used for internal SDK coin representation.
 	DenomPrefix = "ibc"
+
+	// EscrowPrefix is the prefix used for IBC escrow addresses to avoid address collisions
+	// with Public Key addresses.
+	EscrowPrefix = "ibc-v1"
```

Comment:
I would remove these lines; in #7737 "ibc-v1" was given only as an example; it makes total sense to use `Version` instead
```suggestion
```